### PR TITLE
feat: amplifier testnet

### PR DESCRIPTION
**Pull request by talalashraf**
updated testnet.json file with configuration for amplifier 1.0.0 instance deployment on testnet including two evm chains test-avalanche and test-sepolia. same multisig, constaddrdeployer and create3deployers used. updated rpc to lava to avoid rate limits.

### CODE REVIEW

File: axelar-chains-config/info/testnet.json [hunk 1]
```diff
@@ -1,5 +1,226 @@
 {
   "chains": {
+    "test-avalanche": {
```
Comment (milapsheth) [inline axelar-chains-config/info/testnet.json:3]: move it to the bottom since it's a test instance?

File: axelar-chains-config/info/testnet.json [hunk 2]
```diff
@@ -1,5 +1,226 @@
...
+        },
+        "Create3Deployer": {
+          "salt": "Create3Deployer",
+          "address": "0x6513Aedb4D1593BA12e50644401D976aebDc90d8",
+          "deployer": "0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05",
+          "deploymentMethod": "create2",
+          "codehash": "0xf0ad66defbe082df243d4d274e626f557f97579c5c9e19f33d8093d6160808b7",
+          "predeployCodehash": "0x73fc31262c4bad113c79439fd231281201c7c7d45b50328bd86bccf37684bf92"
+        },
+        "InterchainTokenService": {
```
Comment (milapsheth) [inline axelar-chains-config/info/testnet.json:48]: remove it since it was an incorrect deployment

File: axelar-chains-config/info/testnet.json [hunk 3]
```diff
@@ -1,5 +1,226 @@
...
+          "tokenManagerDeployer": "0x8886101bD36e67235c1A0adA586D9F995a1b2Ef9",
+          "interchainToken": "0x706Ba5c25E00d72a9c7AFf6275fCF668507B2EB9",
+          "interchainTokenDeployer": "0x8F568826Cc731b2FAB9EEd0244C6Fe4F57b37aC5",
+          "tokenManager": "0x8807A598D941AEA6cb5b06c78110a6b0F25E3200",
+          "tokenHandler": "0x25AfcfA6B91CFd2370aCE50e68ac63a74F215fBc",
+          "implementation": "0x2a1966E72c2777c7b0E099B474e1Cf6e02132e2e",
+          "predeployCodehash": "0x08a4a556c4db879b4f24104d13a8baf86915d58b12c81b382dfea2a82d2856cf",
+          "address": "0x43AC726252c4A3b362eB4447116deF682e89E20b"
+        },
+        "InterchainTokenFactory": {
```
Comment (milapsheth) [inline axelar-chains-config/info/testnet.json:61]: same

File: axelar-chains-config/info/testnet.json [hunk 4]
```diff
@@ -1,5 +1,226 @@
...
+        "Operators": {
+          "owner": "0xba76c6980428A0b10CFC5d8ccb61949677A61233",
+          "address": "0x6D37Ca166d05132e632C78882391031a3B5b6cE8",
+          "deployer": "0xba76c6980428A0b10CFC5d8ccb61949677A61233",
+          "deploymentMethod": "create2",
+          "codehash": "0xc561dc32ef670c929db9d7fbf6b5f6c074a62a30602481ba3b88912ca6d79feb",
+          "predeployCodehash": "0xc561dc32ef670c929db9d7fbf6b5f6c074a62a30602481ba3b88912ca6d79feb",
+          "salt": "Amplifier Test Temporary"
+        },
+        "InterchainGovernance": {
```
Comment (milapsheth) [inline axelar-chains-config/info/testnet.json:76]: remove it since it's not applicable

File: axelar-chains-config/info/testnet.json [hunk 5]
```diff
@@ -1,5 +1,226 @@
...
+        },
+        "Create3Deployer": {
+          "address": "0x6513Aedb4D1593BA12e50644401D976aebDc90d8",
+          "deployer": "0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05",
+          "deploymentMethod": "create2",
+          "codehash": "0xf0ad66defbe082df243d4d274e626f557f97579c5c9e19f33d8093d6160808b7",
+          "predeployCodehash": "0x73fc31262c4bad113c79439fd231281201c7c7d45b50328bd86bccf37684bf92",
+          "salt": "Create3Deployer"
+        },
+        "AxelarGateway": {
```
Comment (milapsheth) [inline axelar-chains-config/info/testnet.json:134]: same

**Review by milapsheth — APPROVED** [review]

File: axelar-chains-config/info/testnet.json (hunk 1 above)
Comment (talalashraf) [reply axelar-chains-config/info/testnet.json:3]: will remove it soon no ?

**Pull request merged**